### DISCUSSION TITLE
Cant play South America

### DISCUSSION THREAD

**Discussion by Bepovi** [HasangerGames/suroi#348] [General]
How would it be possible to increase the population of the South America server, if there is no option for duos or squads to start the game?

**Comment by adtcqd** [comment]
Me  too, when I play in China, my ping very high, almost 100+ms, when I saw other player want to shoot me, I die soon.(by baidu translation App )
我也一样，当我在中国玩，我的延迟非常高，经常100多毫秒，当我看见人，我已经死了
